content/account_management/faq/_index.md

```diff
@@ -9,6 +9,7 @@ private: true
 * [Are my data and credentials safe?](/account_management/faq/are-my-data-and-credentials-safe)
 * [Help! My Password email never came through!](/account_management/faq/help-my-password-email-never-came-through)
 * [Is it possible to have my login belong to multiple Datadog organizations?](/account_management/faq/is-it-possible-to-have-my-login-belong-to-multiple-datadog-organizations)
+* [Password Requirements](/account_management/faq/password-requirements)
```

Comment:
This is a nitpick, I don't feel too strongly about.

All the other bullet points are questions, with the exception of `Help! My Password email never came through!`. Would it be better to rephrase as:

> What are my password requirements?

Reply:
it's a good nit :)

Reply:
I took the first one => better SEO